app/scripts/directives/processTemplateDialog.js

```diff
@@ -43,6 +68,34 @@
 
     ctrl.$onInit = function() {
       ctrl.loginBaseUrl = DataService.openshiftAPIBaseUrl();
+      ctrl.preSelectedProject = ctrl.selectedProject = ctrl.project;
+      listProjects();
+
+      ctrl.projectEmptyState = {
+        icon: 'pficon pficon-info',
+        title: 'No Project Selected',
+        info: 'Please select a project from the dropdown to load Templates and Images from that project.'
+      };
+
+      ctrl.templatesEmptyState = {
+        icon: 'pficon pficon-info',
+        title: 'No Templates or Images',
+        info: 'The selected project has no templates or images available to import.'
```

Comment:
We'll want to remove "images" from these messages if there are only templates. Maybe

"The selected project has no templates available to instantiate."

(When we add images back, we'll want to say specifically "builder images.")